MekHQ/src/mekhq/gui/adapter/PersonnelTableMouseAdapter.java

```diff
@@ -587,47 +587,56 @@ public void actionPerformed(ActionEvent action) {
                         selected, true);
                 selectedPerson.awardXP(-cost);
                 gui.getCampaign().personUpdated(selectedPerson);
-                // TODO: add personnelTab.getCampaign() report
+                gui.getCampaign().addReport(String.format(resourceMap.getString("gained.format"),
```

Comment:
I think the idea was to add these to the personal log, not the campaign report.

Reply:
I completely misread the log in question for the issue... but was doing this because I wanted to see the changes in the campaign report as I found it rather a bother last night. I'll add an option for the personnel log once I finish the story I'm working on now.